Rubberduck.Parsing/Symbols/DeclarationSymbolsListener.cs

```diff
@@ -136,41 +136,44 @@ private IEnumerable<IAnnotation> FindAnnotations(int line)
         /// </remarks>
         private void DeclareControlsAsMembers(IVBComponent form)
         {
-            if (form.Controls == null) { return; }
-
-            var libraryQualifier = string.Empty;
-            if (_qualifiedModuleName.ComponentType == ComponentType.UserForm)
+            using (var controls = form.Controls)
             {
-                var msFormsLib = _state.DeclarationFinder.FindProject("MSForms");
-                //Debug.Assert(msFormsLib != null);
-                if (msFormsLib != null)
+                if (controls == null) { return; }
+
+                var libraryQualifier = string.Empty;
+                if (_qualifiedModuleName.ComponentType == ComponentType.UserForm)
                 {
-                    // given a UserForm component, MSForms reference is in use and cannot be removed.
-                    libraryQualifier = "MSForms.";
+                    var msFormsLib = _state.DeclarationFinder.FindProject("MSForms");
+                    //Debug.Assert(msFormsLib != null);
```

Comment:
commented code. we explicitly check that assertion now. remove?

Reply:
Done.